x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -465,10 +465,10 @@ export class EndpointDocGenerator extends BaseDataGenerator {
           applied: this.randomChoice(APPLIED_POLICIES),
         },
         configuration: {
-          isolation: false,
+          isolation: this.randomBoolean(),
```

Comment:
not a criticism:

but if these are both random without regard to each other, we will over-represent the incongruent state (where these two values are out-of-sync). Roughly half of the generated endpoints will be in an unhealthy/incongruent isolation state, which may be a rarer occurrence. I don't know how "smart" we want the generator to be, but making these values equal each other the majority (70%) of the time might make for slightly more representative fake-data. If we care about that. Of course left this way it's more likely we cover all edge cases with fewer generated endpoints, so it's not like there's a clear cut thing to do here.

Reply:
I'll just sync them up to the same value for now so that we don't have so many in inconsistent state